flocker/provision/_aws.py

```diff
@@ -24,37 +28,40 @@ def provision_aws(node, package_source, distribution, variants):
         'fedora-20': 'fedora',
         'centos-7': 'centos',
     }[distribution]
-    run(
+
+    commands = []
```

Comment:
These are `post_reboot_commands`? If so, I think that they should be named as such.